This has been submitted upstream; the # TODO (GEN) is a comment used
elsewhere upstream to mark things commented-out for generated files.

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c63ab1f2..73d15fce 100644
--- src/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -56,18 +56,18 @@ endif()
 # Mark files as generated, set compile flags
 set_source_files_properties(${PARSER_SOURCE_DIR}/generated/sqlparser.cpp
     PROPERTIES
-        GENERATED TRUE
+        #TODO(GEN) GENERATED TRUE
         # YYERROR_VERBOSE=1 needed to get a token table for tokenName() even for release builds
         COMPILE_FLAGS "-DYYERROR_VERBOSE=1 ${EXTRA_PARSER_COMPILE_FLAGS} "
 )
-set_source_files_properties(${PARSER_SOURCE_DIR}/generated/sqlparser.h PROPERTIES GENERATED TRUE)
-set_source_files_properties(${PARSER_SOURCE_DIR}/generated/KDbToken.h PROPERTIES GENERATED TRUE)
-set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/KDbConnectionData_sdc.cpp PROPERTIES GENERATED TRUE)
+# TODO(GEN) set_source_files_properties(${PARSER_SOURCE_DIR}/generated/sqlparser.h PROPERTIES GENERATED TRUE)
+# TODO(GEN) set_source_files_properties(${PARSER_SOURCE_DIR}/generated/KDbToken.h PROPERTIES GENERATED TRUE)
+# TODO(GEN) set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/KDbConnectionData_sdc.cpp PROPERTIES GENERATED TRUE)
 
 set_source_files_properties(
     ${PARSER_SOURCE_DIR}/generated/sqlscanner.cpp
     PROPERTIES
-        GENERATED TRUE
+        #TODO(GEN) GENERATED TRUE
         COMPILE_FLAGS "${EXTRA_SCANNER_COMPILE_FLAGS} "
 )
 
